### ISSUE TITLE
[terra-table] terra-table(3.13.0) update breaking ItemCollection.Item

### ISSUE THREAD

**Issue by parvezalisha** [cerner/terra-core#2411]
# Bug Report

## Description
It looks like the latest terra-table(3.13.0) update broke ItemCollection.Item when there is only one element. We have a child element in item without any startAccessory or endAccessory for it. Those get passed as null values to TableRow [here](https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-collection/src/Item.jsx#L140). The table row takes children as an array of [startAccessory, displayContent, comment, endAccessory]. Since three of those are null and TableRow in this [latest change](https://github.com/cerner/terra-core/blob/master/packages/terra-table/src/TableRow.jsx#L71) is trying to clone every child element, we get an error for null elements from React.cloneElement.

## Steps to Reproduce
Create an item collection item as below:
```
    <ItemCollection
      numberOfDisplays={1}
    >
      <ItemCollection.Item>
        <div>
          This is a child element of item
        </div>
      </ItemCollection.Item>
    </ItemCollection>
```

## Expected Behavior
The error shouldn't occur and the above code should be rendered.

## Environment
* Component Version: terra-table 3.13.0
* Browser Name and Version: Chrome
* Node/npm Version [e.g. Node 8/npm 5]: Node 8.9.2, npm 5.5.1
* Webpack Version: ^4.0.0
* Operating System and version (desktop or mobile): Desktop(macOS Sierra v10.12.6)

## Possible Solution
Use React.cloneElement only when child is not null.

<img width="896" alt="Screen Shot 2019-05-22 at 12 13 26 PM" src="https://user-images.githubusercontent.com/10690028/58194398-081e1b80-7c8b-11e9-907b-92eb6977617d.png">

## Mentions
@bjankord

**Comment by papasmile** [comment]
having that same issue.  In mentions it should actually be @bjankord

**Comment by bjankord** [comment]
This should be resolved now in terra-table@3.14.0. Let us know if you continue to see issues with it and we'll work to resolve them.
